### PR TITLE
Feat: add sponsors & banner images upload

### PR DESCRIPTION
**Pull request by Meriem-BM**
Related to https://github.com/Giveth/giveth-dapps-v2/issues/3250

### CODE REVIEW

**Comment by Meriem-BM** [comment]
@mohammadranjbarz @RamRamez for the uploading the image get pinned successfully returning the hash but when I try to open it  (Ex: https://giveth.mypinata.cloud/ipfs/QmVB5Chh4FA3RSqwXj8a83SyuWTMH9DGo9s4r9bhJ1yydg)

I get this error (The owner of this gateway does not have this content pinned to their Pinata account. In order to view this content, please reach out to the owner. - ERR_ID:00006)

But when I upload the same image for example on FE app (update profile pic) it gives me back the same hash and it displays on the browser and on AdminJS.

File: migration/1714018700116-add_archived_QFRound_fields.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    await queryRunner.query(`
+          ALTER TABLE "qf_round" 
+          ADD COLUMN IF NOT EXISTS "bannerBgImage" character varying
+        `);
+
+    await queryRunner.query(`
+          ALTER TABLE "qf_round" 
+          ADD COLUMN IF NOT EXISTS "sponsorsImgs" character varying[] DEFAULT '{}' NOT NULL
+        `);
+  }
```
Comment (CarlosQ96) [inline migration/1714018700116-add_archived_QFRound_fields.ts:16]: You can do the queries in 1 transaction. By adding the next command in the next line

**Review by CarlosQ96 — APPROVED** [review]
LGTM!

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @Meriem-BM  LGTM.
Feel free to merge it.

**Pull request merged**